src/sentry/demo/demo_start.py

```diff
@@ -172,21 +174,28 @@ def get_one_release(org: Organization, project_slug: Optional[str]):
     return f"/organizations/{org.slug}/releases/{version}/?project={project.id}"
 
 
-def get_one_issue(org: Organization, project_slug: Optional[str]):
+def get_one_issue(org: Organization, project_slug: Optional[str], error_type: Optional[str]):
     group_query = Group.objects.filter(project__organization=org)
+    error_type = error_type.lower()
     if project_slug:
         group_query = group_query.filter(project__slug=project_slug)
-    group = group_query.first()
-
+        if error_type:
+            for group in group_query:
```

Comment:
@roggenkemper could you do this with the `filter` function? I think that might be cleaner than using a for loop than breaking.

Reply:
I wasn't able to figure out how to use filter, but I have another idea since I have the function now